scripts/build/Platform/Linux/build_config.json

```diff
@@ -129,7 +129,7 @@
       "CMAKE_OPTIONS": "-G 'Ninja Multi-Config' -DLY_PARALLEL_LINK_JOBS=4",
       "CMAKE_LY_PROJECTS": "AutomatedTesting",
       "CMAKE_TARGET": "all",
-      "CTEST_OPTIONS": "-E (AutomatedTesting::TerrainTests_Main|Gem::EMotionFX.Editor.Tests) -L (SUITE_smoke|SUITE_main) -LE (REQUIRES_gpu) --no-tests=error -T Test",
+      "CTEST_OPTIONS": "-E (AutomatedTesting::TerrainTests_Main) -L (SUITE_smoke|SUITE_main) -LE (REQUIRES_gpu) --no-tests=error -T Test",
```

Comment:
This is great, hopefully terrain tests can be fixed soon in Linux as well and we can remove these hidden exclusion filter.